airflow/providers/google/cloud/log/stackdriver_task_handler.py

```diff
@@ -188,7 +193,7 @@ def read(
         if next_page_token:
             new_metadata['next_page_token'] = next_page_token
 
-        return [messages], [new_metadata]
+        return [((self.task_instance_hostname, messages),)], [new_metadata]
```

Comment:
Indeed... interesting way of reurning data :)

Reply:
Indeed... `List[Tuple[Tuple[str, List]]]`  😄

Reply:
``Tuple[List[Tuple[Tuple[str, str]]], List[Dict[str, str]]]``